ui/shared/ModalPopup.qml

```diff
@@ -5,125 +5,125 @@ import "../imports"
+                visible: !!title
+            }
         }
-      }
 
-      Rectangle {
-          id: closeButton
-          height: 32
-          width: 32
-          anchors.top: parent.top
-          anchors.topMargin: Theme.padding
-          anchors.rightMargin: Theme.padding
-          anchors.right: parent.right
-          radius: 8
+        Rectangle {
+            id: closeButton
+            height: 32
+            width: 32
+            anchors.top: parent.top
+            anchors.topMargin: Theme.padding
+            anchors.rightMargin: Theme.padding
+            anchors.right: parent.right
+            radius: 8
 
-          Image {
-              id: closeModalImg
-              source: "./img/close.svg"
-              anchors.horizontalCenter: parent.horizontalCenter
-              anchors.verticalCenter: parent.verticalCenter
-          }
+            Image {
+                id: closeModalImg
+                source: "./img/close.svg"
+                anchors.horizontalCenter: parent.horizontalCenter
+                anchors.verticalCenter: parent.verticalCenter
+            }
 
-          MouseArea {
-              id: closeModalMouseArea
-              cursorShape: Qt.PointingHandCursor
-              anchors.fill: parent
-              hoverEnabled: true
-              onExited: {
-                  closeButton.color = Theme.white
-              }
-              onEntered:{
-                  closeButton.color = Theme.grey
-              }
-              onClicked : {
-                  popup.close()
-              }
-          }
-      }
-      
-      Separator {
-          id: separator
-          anchors.top: headerContent.bottom
-      }
+            MouseArea {
+                id: closeModalMouseArea
+                cursorShape: Qt.PointingHandCursor
+                anchors.fill: parent
+                hoverEnabled: true
+                onExited: {
+                    closeButton.color = Theme.white
+                }
+                onEntered: {
+                    closeButton.color = Theme.grey
+                }
+                onClicked: {
+                    popup.close()
+                }
+            }
+        }
 
-      Item {
-        id: popupContent
-        anchors.top: separator.bottom
-        anchors.topMargin: Theme.padding
-        anchors.bottomMargin: Theme.padding
-        anchors.left: parent.left
-        anchors.leftMargin: Theme.padding
-        anchors.right: parent.right
-        anchors.rightMargin: Theme.padding
-      }
+        Separator {
+            id: separator
+            anchors.top: headerContent.bottom
+        }
 
-      Separator {
-          id: separator2
-          anchors.bottom: parent.bottom
-          anchors.bottomMargin: 75
-      }
+        Item {
+            id: popupContent
+            anchors.top: separator.bottom
+            anchors.topMargin: Theme.padding
+            anchors.bottom: separator2.top
```

Comment:
This is the only real thing I added. Without this, the content had 0 height (for the contents that relied on `fill: parent`)